extensions/objects/templates/svg-filename-editor.component.ts

```diff
@@ -615,11 +629,10 @@ angular.module('oppia').component('svgFilenameEditor', {
           ctrl.polyOptions.bboxPoints.push(
             new polyPoint(startPt.x, startPt.y));
         }
-        var size = ctrl.fabricjsOptions.size;
         var shape = new fabric.Polyline(ctrl.polyOptions.bboxPoints, {
           fill: ctrl.fabricjsOptions.fill,
           stroke: ctrl.fabricjsOptions.stroke,
-          strokeWidth: parseInt(size.substring(0, size.length - 2)),
+          strokeWidth: getStrokeWidth(),
           strokeUniform: true,
           strokeLineCap: 'round'
         });
```

Comment:
The method `createPolyShape` may need more comments. E.g. why remove from canvas un L644? Why do `object.selectable = true;` in L652?

Reply:
Done.